src/sagas/ChangePasswordToken.saga.ts

```diff
@@ -42,9 +42,17 @@ function* ChangePasswordToken(action: any): SagaIterator {
     yield call(delay, 300)
     yield put(Actions.ChangePasswordToken.onChangePasswordTokenClearError())
 
-    toast.error(e, {
-      className: 'toast',
-      autoClose: 2500,
-    })
+    try {
+      const { message } = JSON.parse(e)
+      toast.error(message, {
```

Comment:
Shouldn't be calling these functions from inside the saga. Should be using `call` and these changes should have unit tests.